### ISSUE TITLE
Platforms containing only numbers (ie ASA 5500) being converted to int causing failure

### ISSUE THREAD

**Issue by matthewpomeryssq** [community-fabric/ipfabric-ansible#25]
When attempting to group by platform in the ipfabric inventory module:
ie: 
keyed_groups:
  - key: platform
    prefix: "p_"
    separator: ""

A failure occurs if any of the device platform values contain only numbers:
ie:
platform=5500

The error generated is:
* Failed to parse inventory.yml with auto plugin: Invalid group name format, expected a string or a list of them or
dictionary, got: <class 'int'>
  File "/usr/local/lib/python3.8/site-packages/ansible/inventory/manager.py", line 290, in parse_source
    plugin.parse(self._inventory, self._loader, source, cache=cache)
  File "/usr/local/lib/python3.8/site-packages/ansible/plugins/inventory/auto.py", line 59, in parse
    plugin.parse(inventory, loader, path, cache=cache)
  File ".../ansible_collections/community_fabric/ansible/plugins/inventory/inventory.py", line 156, in parse
    self.main()
  File ".../ansible_collections/community_fabric/ansible/plugins/inventory/inventory.py", line 145, in main
    self._populate(self.devices_list)
  File ".../ansible_collections/community_fabric/ansible/plugins/inventory/inventory.py", line 139, in _populate
    self._add_host_to_keyed_groups(self.get_option("keyed_groups"), new_vars, to_text(hostname), strict=strict)
  File "/usr/local/lib/python3.8/site-packages/ansible/plugins/inventory/__init__.py", line 443, in _add_host_to_keyed_groups
    raise AnsibleParserError("Invalid group name format, expected a string or a list of them or dictionary, got: %s" % type(key))

I suspect the values returned by the query must be type cast as strings.

**Comment by minitriga** [comment]
Fixed the issue, installed 0.0.2 from ansible galaxy using `ansible-galaxy collection install community_fabric.ansible` and it should work. 

Let me know if there are any issues. 

```
    "p_5500": {
        "hosts": [
            "HWLAB-FW-C5510"
        ]
    },
    "p_asav": {
        "hosts": [
            "L1FW1",
            "L35FW1",
            "L35FW2",
            "L68ASAV5",
            "L68ASAV6"
        ]
    }
}
```

**Comment by matthewpomeryssq** [comment]
Looks good!
